Server/src/routes/DimensionsRouter.ts

```diff
@@ -12,15 +12,15 @@ let dimensionsController = new DimensionsController();
  * **IMPORTANT**
  * These routes are temporarily unprotected, but later need to be uncommented
  */
-router.post('/api/v1/dimensions', [JWTAuthenticator.verifyJWT, JWTAuthenticator.verifyRoot], (request: Request, response: Response) => {
+router.post('/api/v1/dimensions', (request: Request, response: Response) => {
```

Comment:
why arent post and put protected? is it mean like that?

Reply:
That must have been a merge conflict. They are supposed to be protected.